account_invoice_user_authorization_payment/views/account_move_views.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+
+    <record id="user_authorization_payment_tree" model="ir.ui.view">
+        <field name="name">account.move.tree (User Authorization Payment)</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_in_invoice_tree"/>
+        <field name="groups_id" eval="[(4, ref('account.group_account_user'))]"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='name']" position="after">
+                <field name="create_uid" />
```

Comment:
I miss some things in this view:
- `user_authorization_payment` field is not shown.
- Both create user and payment user should be set as `optional="show"`.
- Inheriting in this view makes these field visible for both customer and vendor invoices. If you look at which view is loaded in case of vendor bills, you can find out that is another view, only for vendor bills. Then, if you inherit from it, these fields won't be shown for customer invoices (and that's we should need).